src/features/game/expansion/components/travel/IslandTravelModal.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
```

Comment:
Just moved this out of `IslandTravel` and into a modal component so I could add the tutorial.